cypress/e2e/09-multiplex_testing.cy.js

```diff
@@ -86,6 +86,7 @@ describe("Testing with multiplex devices", () => {
       );
       cy.get('input[name="inconclusive-tests"]')
         .should("not.be.checked")
+        .should("be.enabled")
```

Comment:
omg I have been wondering for a long time what was missing in this check and never occurred to me to check the enable prop! Nice